### PR TITLE
fix `access-mapping.yml` extra_facets title

### LINKED ISSUE #2483: An issue in ACCESS-ESM extra_facets
**Describe the bug**
In file (https://github.com/ESMValGroup/ESMValCore/tree/main/esmvalcore/config/extra_facets/access-mappings.yml)

the title `ACCESS_ESM` should be changed to `ACCESS-ESM1-5`.

**Please attach**
  - The recipe that you are trying to run, you can find a copy in the `run` directory in the output directory
  - The `main_log_debug.txt` file, this can also be found in the `run` directory in the output directory

### PR DESCRIPTION
**Pull request by rhaegar325**
## Description
make a slight change in title of `access-mapping.yml` to avoid a potential bug

Closes #2483 

Link to documentation:

***

## [Before you get started](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#getting-started)

- [x] [☝ Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do

## [Checklist](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#checklist-for-pull-requests)

It is the responsibility of the author to make sure the pull request is ready to review. The icons indicate whether the item will be subject to the [🛠 Technical][1] or [🧪 Scientific][2] review.

[1]: https://docs.esmvaltool.org/en/latest/community/review.html#technical-review
[2]: https://docs.esmvaltool.org/en/latest/community/review.html#scientific-review

- [x] [🧪][2] The new functionality is [relevant and scientifically sound](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#scientific-relevance)
- [x] [🛠][1] This pull request has a [descriptive title and labels](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-title-and-label)
- [x] [🛠][1] Code is written according to the [code quality guidelines](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#code-quality)
- [x] [🧪][2] and [🛠][1] [Documentation](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#documentation) is available
- [x] [🛠][1] [Unit tests](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#tests) have been added
- [x] [🛠][1] Changes are [backward compatible](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#backward-compatibility)
- [x] [🛠][1] Any changed [dependencies have been added or removed](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#dependencies) correctly
- [x] [🛠][1] The [list of authors](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#list-of-authors) is up to date
- [x] [🛠][1] All [checks below this pull request](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-checks) were successful

***

To help with the number pull requests:

- 🙏 We kindly ask you to [review](https://docs.esmvaltool.org/en/latest/community/review.html#review-of-pull-requests) two other [open pull requests](https://github.com/ESMValGroup/ESMValCore/pulls) in this repository

### CODE REVIEW

**Comment by rbeucher** [comment]
Thanks @rhaegar325. Why this was not picked by the tests?

**Comment by rbeucher** [comment]
Tests are failing. @rhaegar325 please have a loop as soon as possible.

**Comment by rbeucher** [comment]
Also. Can you explain the bug in more details, please? for the records

**Comment by rbeucher** [comment]
@rhaegar325 We usually use `dataset: ACCESS-ESM1-5` not `ACCESS_ESM1_5`

**Comment by rhaegar325** [comment]
@rbeucher, now this error has been fixed. The bug is caused by we deside to change the file name of our cmoriser from `access_esm.py` to `access_esm1_5.py`. but I did not change the title in  extra_facets `access-mapping.yml`, so when we use the cmoriser it can not load the extra_facets. Also, in `test_access_esm1_5.py`, I did not change the dataset name tp `ACCESS-ESM1-5`, so the tast didn't pick this error up. silly mistake, won't happen again.

File: tests/integration/cmor/_fixes/access/test_access_esm1_5.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ def test_only_latitude(monkeypatch, cubes_2d):
 
 def test_only_longitude(monkeypatch, cubes_2d):
     """Test fix."""
-    fix = get_fix_allvar('Amon', 'mon', 'tas')
+    fix = get_fix_allvar('Amon', 'mon', 'pr')
```
Comment (rbeucher) [inline tests/integration/cmor/_fixes/access/test_access_esm1_5.py:214]: Why did you change the test to `pr`. Is it not working for `tas`?

Comment (rhaegar325) [reply tests/integration/cmor/_fixes/access/test_access_esm1_5.py:214]: since we have a Tas class, so `tas` will go through class Tas and AllVars, so we have a test especially for `tas` below, in this part I think it's better to have a general variable which only go through `AllVars` for those coordinates test.

**Review by rbeucher — APPROVED** [review]
